codecov.yml

```diff
@@ -0,0 +1,57 @@
+#
```

Comment:
is the filename should be `.codecov.yml`?

Reply:
both should work but yes, let me rename it for consistency with the other config files